### ISSUE TITLE
how to get validation label and predict image for all after training

### ISSUE THREAD

**Issue by aipyoon** [ultralytics/yolov5#11135]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I've found a similar question and got to check run\tran\exp\jpg : https://github.com/ultralytics/yolov5/issues/4269
but my question is how to see all predicted result of all validation data as images, not a part of validation data.

I'm not sure if it is related to batch size because I set 16 batch size and got 16 validation images.
I'm going to check the accuracy of trained model by identifying the error between ground truth and predicted result for all validation dataset to enhance training datasets.

Is there any way to solve this question? 

Many thanks 

****this is the validation image for 16 data (train 128 images, validation 32images)
![val_batch0_pred](https://user-images.githubusercontent.com/127379275/223940150-f6cbf6b8-d9e9-4b88-b24d-e1c94d0491dc.jpg)

**Comment by glenn-jocher** [comment]
@aipyoon hi there!

After training, to get predictions for all of your validation data, you'll want to run inference on your trained model using the validation dataset. 

You can use the 'detect.py' script to make predictions on a specified dataset using a trained model. 
You'll find the instructions on how to do this in the 'infer' section of the documentation at https://docs.ultralytics.com/yolov5/.

Feel free to reach out if you have any more questions!
